### PR TITLE
DevWs for NIE-TECH CO., LTD. containing containing Aeotec Wallmote

### PR DESCRIPTION
**Pull request by sky-nie**

### CODE REVIEW

**Comment by sky-nie** [comment]
We have repeatedly tested and found that after modifying the fingerprint, Minoston Wallmote is more likely to succeed when pairing to this DTH.

File: devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy [hunk 1]
```diff
@@ -27,7 +27,7 @@ metadata {
         fingerprint mfr: "0086", model: "0081", deviceJoinName: "Aeotec Remote Control", mnmn: "SmartThings", vid: "generic-2-button" //Aeotec Wallmote
         fingerprint mfr: "0060", model: "0003", deviceJoinName: "Everspring Remote Control", mnmn: "SmartThings", vid: "generic-2-button" //Everspring Wall Switch
         fingerprint mfr: "0371", model: "0016", deviceJoinName: "Aeotec Remote Control", mnmn: "SmartThings", vid: "generic-2-button" //Aeotec illumino Wallmote 7
-        fingerprint mfr: "0312", model: "D001", deviceJoinName: "Minoston Remote Control", mnmn: "SmartThings", vid: "generic-4-button" //Minoston Wallmote
+        fingerprint model: "D001", mfr: "0312", prod: "0924", deviceJoinName: "Minoston Remote Control", mnmn: "SmartThings", vid: "generic-4-button" //Minoston Wallmote
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy]: Generally all fingerprints follow this order:
`mfr: "...", prod: "...", model: "..."`
so please, adjust yours.

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy (hunk 1 above)
Comment (sky-nie) [reply devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy]: OK, I modified it.

**Review by PKacprowiczS — APPROVED** [review]

**Comment by k-morawski** [comment]
Dear @sky-nie , please let me know if this request is a new integration request or this is an update in DevWS of your previous MR40Z integration which is already approved. It would be better to request an update to existing integration. Otherwise in the catalog there will be two entries for the same device.
What's more there is a typo in pairing instructions: "Press the first button 5 times, until the LED indicator blink bue." bue should be probably replaced with blue. Could you please check it? Many thanks

**Comment by sky-nie** [comment]
This is an update in DevWS of my previous MR40Z integration which is already approved. Now the DTH is updated, but I can't find the place to start the update request in the Developer Workspace (for the released product, if the product has not been modified), so I have to modify the product information  to start the update request.

**Comment by PKacprowiczS** [comment]
Closing this PR, as partner canceled DevWS request connected to it.

**Pull request closed without merging**